common/app/model/dotcomrendering/DotcomRenderingDataModel.scala

```diff
@@ -165,6 +167,7 @@ object DotcomRenderingDataModel {
         "matchUrl" -> model.matchUrl,
         "matchType" -> model.matchType,
         "isSpecialReport" -> model.isSpecialReport,
+        "newsletterToEmbed" -> model.newsletterToEmbed,
```

Comment:
I see your point! `newsletterToEmbed` isn't a great name, but `newsletter` might be a bit too abstract - in future we might want to have another field (with the same data type) to say the article is actually about the newsletter rather than just supposed to include the sign-up block for it. How about `newsletterToPromote` ?

Reply:
Or `promotedNewsletter` not fused either way!

Reply:
👍 changed to `promotedNewsletter`